ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2389,12 +2389,19 @@ router_isis:
   instance: <ISIS Instance Name>
   net: < CLNS Address to run ISIS | format 49.0001.0001.0000.0001.00 >
   router_id: < IPv4_address >
+  is_type: < level-1 | level-1-2 | level-2 >
   log_adjacency_changes: < true | false >
+  mpls_ldp_sync_default: < true | false >
+  timers:
+    local_convergence:
+      protected_prefixes: < true | false >
+      delay: < number of milliseconds (Optional, default is 10000) >
+  advertise_passive_only: < true | false >
   no_passive_interfaces: < List no-passive-interface >
-  is_type: < level-1 | level-1-2 | level-2 >
   address_family: < List of Address Families >
   isis_af_defaults:
```

Comment:
This implementation should be refactored into the separate address families. We should not do this kind of abstraction in `eos_cli_config_gen`. Since it was already there before this PR, we need to keep it until 4.0. I have submitted issue #1299 to refactor in 4.0.

Reply:
Added new data model and molecule example to address this.